src/components/carousel-section/carousel-section.tsx

```diff
@@ -42,18 +42,24 @@ export function CarouselSection({ title, type }: CardCarouselSectionProps) {
           {recentPropertiesResult.data?.recentProperties && (
             <Carrousel>
               {recentPropertiesResult.data.recentProperties.map((property: Property) => (
-                <CardProperties
-                  img={property.imageUrls[0]}
+                <Link
+                  to={`/detalhes-do-imovel/${property.id}`}
```

Comment:
poderia colocar essas rotas comuns da aplicacao numa constante separada, para nao correr o risco de escrever alguma letra errada e bugar

```suggestion
                  to={`${appRoutes.imobleDetails}/${property.id}`}
```